### ISSUE TITLE
An element has a name of “section-nav” which is a deprecated way of making an anchor target. Id should be used instead

### ISSUE THREAD

**Issue by ollie-iterators** [CityOfBoston/Iterators#205]
Screengrab/videos (insert file names here) 
2020 11 10, Element has deprecated way of making an anchor target.mp4 
[2020 11 10, Element has deprecated way of making an anchor target.mp4.zip](https://github.com/CityOfBoston/Iterators/files/5524941/2020.11.10.Element.has.deprecated.way.of.making.an.anchor.target.mp4.zip)
 

Link 

https://d8-stg.boston.gov/departments/boston-centers-youth-families/bcyf-mirabella-pool 

 

Node # (if known) 

4266 

 

Type of User:  Anonymous 

 

Environment 

(Device > OS > Browser) 

Computer > MacOS 10.15.7 > Chrome 86 

 

Trusted Tester Test 

Links 

 

Brief Description 

An element has a name of “section-nav” which is a deprecated way of making an anchor target. Id should be used instead 

 
Steps to reproduce 

00:00, Load the URL 

00:18, The element that is using the name of “section-nav” is found on the page. It is the pink square that is flashing near the section navigation links (Pool Procedures”, “Rates and Schedules”, and “More Resources”) 

 

Desired Result 

No links have deprecated ways of making an anchor target 

 

Actual Result 

A link has a deprecated way of making an anchor target (name = section-nav) 

Note: Nodes that also have this error: 21, 45361, 7396, 21311, 121, 64966

**Comment by ollie-iterators** [comment]
This has been verified
